forms-web-app/src/controllers/eligibility.js

```diff
@@ -21,14 +21,15 @@ exports.postDecisionDate = (req, res) => {
 
   const date = moment(`${year}-${month}-${day}`, 'Y-M-D', true);
 
-  if (!date.isValid()) {
+  const currentDate = moment();
+
+  if (!date.isValid() || date.isAfter(currentDate)) {
     errors.push({ 'decision-date': 'Invalid date' });
   }
 
-  if (errors) {
+  if (errors.length) {
```

Comment:
Only just thought this, but there's no guarantee that `req.body.errors` will be an error - if someone send in a body with `{ errors: true }` then this will fail (presumably with a 500).

Are we ok with this or do we want some for of `if (!Array.isArray(errors)) { /* do something */ }`?

Reply:
Happy to change that now.